server/netlink_handler.go

```diff
@@ -65,7 +65,7 @@ func (s *server) ConnectBridge(ctx context.Context, req *pb.ConnectBridgeRequest
 		}, err
 	}
 
-	hostVethName := utils.GenerateVethName(req.PodUUID)
+	hostVethName := utils.GenerateVethName(req.PodUUID + req.ContainerVethName)
```

Comment:
Is there any concern?

Reply:
Fix the OVS side link problem that all the veth's name is same.